src/Imghoard.Tests/ClientTests.cs

```diff
@@ -10,9 +10,29 @@ public class ClientTests
         [Fact]
         public void ConstructClientTest()
         {
-            var i = new ImghoardClient(ImghoardClient.Config.Default());
+            var i = new ImghoardClient(Config.Default());
```

Comment:
Could we make it so that if you pass null it will fallback to null? this way we could make it optional to pass a Config.

Reply:
By default, if the constructor is empty, it uses the default configuration, this was just a straggler that was oversighted from testing.